src/bundle/Controller/ContentViewController.php

```diff
@@ -111,16 +111,21 @@ public function versionsTabAction(
         return $view;
     }
 
-    public function locationsTabAction(ContentView $view, UiLocationService $uiLocationService)
-    {
+    public function locationsTabAction(
```

Comment:
As for the versions pull-request, I'd move the action to the `LocationController`.